js/bitvavo.js

```diff
@@ -1450,10 +1450,10 @@ module.exports = class bitvavo extends Exchange {
             };
         }
         let type = undefined;
-        if ('success' in transaction) {
+        if ('success' in transaction || 'address' in transaction) {
```

Comment:
@kroitor this was the only difference I spot between deposits and withdrawals. Withdrawals have an address attached.